### PR TITLE
Bump to v1.0.0rc4

### PR DESCRIPTION
**Pull request by VVsssssk**
Thanks for your contribution and we appreciate it a lot. The following instructions would make your pull request more healthy and more easily get feedback. If you do not understand some items, don't worry, just make the pull request and seek help from maintainers.

## Motivation

Bump to v1.0.0rc4.

## Modification

Please briefly describe what modification is made in this PR.

## BC-breaking (Optional)

Does the modification introduce changes that break the back-compatibility of the downstream repos?
If so, please describe how it breaks the compatibility and how the downstream projects should modify their code to keep compatibility with this PR.

## Use cases (Optional)

If this PR introduces a new feature, it is better to list some use cases here, and update the documentation.

## Checklist

1. Pre-commit or other linting tools are used to fix the potential lint issues.
2. The modification is covered by complete unit tests. If not, please add more unit test to ensure the correctness.
3. If the modification has potential influence on downstream projects, this PR should be tested with downstream projects.
4. The documentation has been modified accordingly, like docstring or example tutorials.

### CODE REVIEW

**Comment by filaPro** [comment]
#1548 should probably not been mentioned in changelog as it was reverted in #1556.

File: docs/en/changelog.md [hunk 1]
```diff
@@ -1,5 +1,34 @@
...
+- Support [FCAF3D](https://arxiv.org/pdf/2112.00322.pdf)
+
+#### New Features
+
+- Support [FCAF3D](https://arxiv.org/pdf/2112.00322.pdf) (#1547)
+- Add the transformation to support multi-camera 3D object detection (#1580)
+
+#### Improvements
+
+- Remove limit the maximum number of points during SUN RGB-D preprocessing (#1555)
```
Comment (Tai-Wang) [inline docs/en/changelog.md:16]: Remove the limitation of the maximum number of points xxxx

File: docs/en/changelog.md [hunk 2]
```diff
@@ -1,5 +1,34 @@
...
+- Support circle ci (#1647)
```
Comment (Tai-Wang) [inline docs/en/changelog.md:17]: ci -> CI

File: docs/en/changelog.md [hunk 3]
```diff
@@ -1,5 +1,34 @@
...
+- Add mim to extras_require in setup.py (#1560, #1574)
+
+#### Bug Fixes
+
+- Flip yaw angle for DepthInstance3DBoxes.overlaps (#1556)
```
Comment (Tai-Wang) [inline docs/en/changelog.md:22]: No need to list this revert commit

File: mmdet3d/__init__.py [hunk 4]
```diff
@@ -19,7 +19,7 @@ def digit_version(version_str):
 
 
 mmcv_minimum_version = '1.4.8'
-mmcv_maximum_version = '1.6.0'
+mmcv_maximum_version = '1.7.0'
```
Comment (Tai-Wang) [inline mmdet3d/__init__.py:22]: Should also modify the FAQ and other places accordingly

**Review by Tai-Wang — COMMENTED** [review]
Please add commits that are directly made to the master branch.

**Review by Tai-Wang — APPROVED** [review]

**Pull request merged**